src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/executor/ExtractExecutorImpl.java

```diff
@@ -126,19 +126,38 @@ private static void maybeAddTimeRange(
 
   @Override
   public int run(Arguments arguments) throws SQLException, IOException {
+    Preconditions.checkArgument(
```

Comment:
I may missed that, but I didn't notice tests for this case. Could you add it?

Reply:
Done.